Networking/NetworkingTests/Remote/SitePluginsRemoteTests.swift

```diff
@@ -64,4 +64,142 @@ class SitePluginsRemoteTests: XCTestCase {
             XCTAssertNotNil(error)
         }
     }
+
+    // MARK: - Install plugin tests
+
+    /// Verifies that installPlugin properly parses the sample response.
+    ///
+    func test_installPlugin_properly_returns_plugins() {
```

Comment:
super nit: it's expected to return a single plugin?
```suggestion
    func test_installPlugin_properly_returns_plugin() {
```